system-test/test.sh

```diff
@@ -29,7 +29,7 @@ cp -r "$PWD/system-test/busybench" "$TESTDIR"
 cd "$TESTDIR/busybench"
 
 retry npm install pify @types/pify typescript gts @types/node >/dev/null
-retry npm install --nodedir="$NODEDIR" "$PROFILER" >/dev/null
+retry npm install --nodedir="$NODEDIR" ${BINARY_HOST:+--pprof_binary_host_mirror=$BINARY_HOST}  "$PROFILER" >/dev/null
```

Comment:
Nit: you wrapped the lines in system_test.sh to fit them in 80 columns, but here lines are long. Be consistent?

Reply:
Done.